plenum/bls/bls_key_register_pool_manager.py

```diff
@@ -21,12 +25,18 @@ def get_key_by_name(self, node_name, pool_state_root_hash=None):
         if self._current_pool_state_root_hash != pool_state_root_hash:
             self._current_pool_state_root_hash = pool_state_root_hash
             self._load_keys_for_root(pool_state_root_hash)
-
+        if not self._pool_manager.config.VALIDATE_SIGN_WITHOUT_BLS_KEY_PROOF and \
+                self._current_bls_keys_proof.get(node_name, None) in [False, None]:
+            logger.info("{} has no proof of possession for BLS public key.".format(node_name))
```

Comment:
Maybe this is better to write a warning in `_load_keys_for_root`, so no need to have any changes here? `self._current_bls_keys.get(node_name)` will just return None

Reply:
Moved check to _load_keys_for_root.